backend/ches/Exceptions/ChesException.cs

```diff
@@ -32,7 +32,7 @@ public class ChesException : HttpClientRequestException
         /// <param name="client"></param>
         /// <param name="model"></param>
         public ChesException(HttpClientRequestException exception, IHttpRequestClient client, ErrorResponseModel model)
-            : this($"{exception.Message}{Environment.NewLine}", exception, exception.StatusCode)
+            : base($"{exception.Message}{Environment.NewLine}", exception, exception.StatusCode ?? HttpStatusCode.InternalServerError)
```

Comment:
It's technically impossible to have the situation where the `exception.StatusCode == null`, but `HttpClientRequestException` inherits a nullable `StatusCode` property.